demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,251 @@
+
+For example, in a UNIX or Linux computer (and later in Macs), the user might type `ls` to list the files in a particular location.  In MS-DOS (and later in Windows), a user might type `dir` to do the same thing.
+
+Here's a screenshot of a very common thing you might want to do: list files in a particular directory.  In the image, the user is working in a directory called `bias_variance_tradeoff` and asks the computer to list the files in that directory using the command `ls`.  Three things are listed: two files, and a directory called `media`.  The user then asks for a list of the files in the `media` directory by typing `ls media`, and six file names are shown.
+
+![A terminal window in which the user is working in a directory called `bias_variance_tradeoff`.  The command `ls` is followed by a file listing of two files, and a directory called `media`.  The next command is `ls media`, and six file names are shown.](media/ls.png)
+
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+You may have never used a CLI, and that's perfectly fine.  It doesn't mean you're less skilled or intelligent than people who do use the command line.  
+
+This is important to remember because sometimes in tech there are **gatekeepers**, people who think their way to do something is the only or best way, and that if you don't do things their way, you're wrong or undeserving.  For example, you might run across people who use the command line frequently and consider people who don't use the command line to be less talented.  That can be a form of gatekeeping!
+
+**Please don't let gatekeepers change your understanding of your own competence!** 
+
+</div>
+
+## Command Line Interface Common Questions
+
+If you've never used a command line, you might have a few common questions, like:
+
+* How do I get to the command line?  What program do I open in my computer to be able to type in commands?
+* What do I type into the command line to do things?  How will I know what to type?
+* What will happen if I get things wrong?
+
+Where is the Command Line Interface?
+----
+
+Most people use a Mac or a Windows operating system on their computer.  To get to the command line, you'll use **Terminal** in macOS or **Command Prompt** or **PowerShell** or **Git Bash** in Windows.  Using the command line can be a little different based on the computer system you use and which program you use. Terminal is a Linux-like environment that uses the bash language, while Command Prompt on Windows is limited to MS-DOS commands without scripting abilities.  PowerShell is an automation engine designed for Microsoft products that you can also run commands from, but we generally don't recommend it for working with Git.  And Git Bash is a Windows add-on that allows Windows users to have the same powerful and industry-standard Linux-like interface that Mac users get by default.
```

Comment:
This content feels a bit too jargon-y advanced for folks coming in with zero idea of what a CLI is and just wanting a general overview. Consider rewriting to be a bit more general/conceptual?

Reply:
Alternatively, cosider adjusting the module scope to include this if you're wanting to keep it. This feels enough like "significant knowledge" to be in the learning objectives.

Reply:
I disagree that it's too advanced ... a big part of demystifying is "but what do I click on to do this thing?".  If you don't know where in your computer you can do a thing, it's still very mysterious.  But I will think about how to integrate this a bit more smoothly in the scope.

Reply:
Actually I'm going to simplify this a lot and just mention the programs and not what differentiates them.  Tell me what you think!